push_live.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+TMP=/tmp/telemetry-push
+
+rm -rf $TMP
```

Comment:
when removing paths, better to use quotes always, even though this knows the path is not going to have sneaky whitespace

Reply:
agreed, but, this is a crappy little nothing of a script for such care.  I know, I know.